### DISCUSSION TITLE
Hide specific env variables

### DISCUSSION THREAD

**Discussion by axi92** [in-toto/witness#510] [Q&A] [answered]
Can I hide specific env variables from the attestation environment?
All sensitive build server envs are postfixed with _PASSWORD or _SECRET can I omit those from the attestation?

**Comment by jkjell** [comment, marked as answer]
Great question! We will have better support for this soon. You can get a preview [here](https://github.com/in-toto/go-witness/pull/355). For the list of what's blocked today, here's where you can find the [list](https://github.com/in-toto/go-witness/blob/main/attestation/environment/blocklist.go).

**Reply by axi92** [reply]
I saw that the CI system Bamboo from Atlassian is not in there, should I comment on the PR or just drop it here?

https://github.com/Puliczek/awesome-list-of-secrets-in-environment-variables/issues/12

**Reply by kairoaraujo** [reply]
Hi @axi92,
Thank you for your collaboration.
Yes, it is always good to comment :)
